package-lock.json

```diff
@@ -3238,6 +3238,17 @@
           "requires": {
             "ansi-regex": "^4.1.0"
           }
+        },
+        "write-file-atomic": {
```

Comment:
What's up with these package-lock changes?

Reply:
Looks like its required by "@jest/transform"

Reply:
Nothing productive. I updated it.